File: server/src/main/java/com/rnazarapps/server/model/Message.java
package com.rnazarapps.server.model;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
public class Message {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String text;
    private Long authorId;
    private LocalDateTime timestamp;

    // Getters and setters
}

File: app/src/main/java/com/rnazarapps/commsandtaskmanageapp/model/Message.kt
package com.rnazarapps.commsandtaskmanageapp.model

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

// Клас для повідомлень
data class Message(
    @SerializedName("id") val id: Long,
    @SerializedName("textChannelId") val textChannelId: Long,
    @SerializedName("authorId") val authorId: Long,
    @SerializedName("text") val text: String,
    @SerializedName("timestamp") val timestamp: LocalDateTime
)